resources/wrapper/godelw

```diff
@@ -161,23 +164,35 @@ GODEL_BASE_DIR=${GODEL_HOME:-$HOME/.godel}
 # determine OS
 OS=""
 EXPECTED_CHECKSUM=""
-case "$(uname)" in
-    Darwin*)
+case "$(uname)-$(uname -m)" in
+    Darwin-x86_64)
         OS=darwin
-        EXPECTED_CHECKSUM=$DARWIN_CHECKSUM
+        ARCH=amd64
+        EXPECTED_CHECKSUM=$DARWIN_AMD64_CHECKSUM
         ;;
-    Linux*)
+    Darwin-aarch64)
+        OS=darwin
+        ARCH=arm64
+        EXPECTED_CHECKSUM=$DARWIN_ARM64_CHECKSUM
+        ;;
+    Linux-x86_64)
+        OS=linux
+        ARCH=amd64
+        EXPECTED_CHECKSUM=$LINUX_AMD64_CHECKSUM
+        ;;
+    Linux-aarch64)
```

Comment:
Can you verify/confirm that output of `uname -m` on the linux arm environment you are running returns `aarch64`?

Reply:
Yes it does, namely my ubuntu 20.04 on raspberry pi 

```sh
ruckc@pi50:~$ uname
Linux
ruckc@pi50:~$ uname -m
aarch64
ruckc@pi50:~$ uname -a
Linux pi50 5.4.0-1032-raspi #35-Ubuntu SMP PREEMPT Fri Mar 19 20:52:40 UTC 2021 aarch64 aarch64 aarch64 GNU/Linux

```